src/game/boe.dlgutil.cpp

```diff
@@ -1170,13 +1219,25 @@ void handle_talk_node(int which_talk_entry) {
 				save_talk_str1 = s1 >= 0 ? univ.scenario.spec_strs[s1] : "";
 				save_talk_str2 = s2 >= 0 ? univ.scenario.spec_strs[s2] : "";
 			}
-			get_strs(save_talk_str1, save_talk_str2, eSpecCtxType::SCEN, s1, s2);
+			univ.get_strs(save_talk_str1, save_talk_str2, eSpecCtxType::SCEN, s1, s2);
 			put_pc_screen();
 			put_item_screen(stat_window);
 			break;
 	}
-	
-	place_talk_str(save_talk_str1,save_talk_str2,0,dummy_rect);
+
+	if(false){
+		RECORD_WHICH_NODE:
+			if(!is_redo) dialog_history.push_back({false, true, which_talk_entry, {"", ""}});
+			place_talk_str(save_talk_str1,save_talk_str2,0,dummy_rect);
+			reset_talk_words();
+			return;
+	}
+
+	FINISH_TALK_NODE:
+		dialog_history.push_back({can_save_talk, false, -1, {save_talk_str1, save_talk_str2}});
+		place_talk_str(save_talk_str1,save_talk_str2,0,dummy_rect);
+		reset_talk_words();
+		return;
```

Comment:
This stuff is still indented one level too deep (except for `if(false){` and `}`).

Reply:
whoops, sorry.